File: njange/fizzbuzz.py
# Loop through numbers from 1 to 100 
for i in range(1, 101): 
    # Check if the number is divisible by both 3 and 5 
    if i % 3 == 0 and i % 5 == 0: 
        print("FizzBuzz") # Print "FizzBuzz" for multiples of both 3 and 5 
    # Check if the number is divisible by 3 
    elif i % 3 == 0: 
        print("Fizz") # Print "Fizz" for multiples of 3 
    # Check if the number is divisible by 5 
    elif i % 5 == 0: 
        print("Buzz") # Print "Buzz" for multiples of 5 
    else: 
        print(i) # Print the number if it is not divisible by 3 or 5

File: wathika-eng/fizz_buzz.go
// Package main will solve the FizzBuzz problem
package main

import "fmt"

func Fizz() {
	// a loop which runs from 0 to 100
	for a := 0; a <= 100; a++ {
		// if a is divisible by both 3 and 5, print fizzbuzz
		if a%3 == 0 && a%5 == 0 {
			fmt.Printf("FizzBuzz\n")
		} else if a%3 == 0 { // if a is divisible by 3 only, print Fizz
			fmt.Printf("Fizz\n")
		} else if a%5 == 0 { // if a is divisible by 5 only, print Buzz
			fmt.Printf("Buzz\n")
		} else { // if not divisible by either 3 or 5, just print the value
			fmt.Printf("%v\n", a)
		}
	}
}

func main() {
	Fizz()
}
